costumes.js

```diff
@@ -0,0 +1,110 @@
+const createCostume = async (
+    pool,
+    {
+        name, 
+        category, 
+        gender, 
+        size, 
+        type, 
+        stockCount, 
+        price
+    }) => {
+    const {rows:[costume]} = await pool.query(`
+        INSERT INTO costumes(
+            name,
+            category,
+            gender,
+            size,
+            type,
+            stock_count,
+            price
+        ) VALUES ($1, $2, $3, $4, $5, $6, $7)
+        RETURNING *;   
+    `, [name, category, gender, size, type, stockCount, price]);
+    return costume;
+}
+
+const getAllCostumes = async (pool) => {
+    const {rows: costumes} = await pool.query(`
+        SELECT * FROM costumes;
+    `)
+    return costumes;
+}
+
+const getCostumeById = async (pool, id) => {
+    const {rows:[costume]} = await pool.query(`
+        SELECT *
+        FROM costumes
+        WHERE id = $1;
+    `, [id])
+    if (costume === undefined) {
```

Comment:
Will it equal `undefined` or `null`? FWIW I think it would be reasonable to just do a truthy check in this kind of situation (since null or undefined would both be unexpected states)
```suggestion
    if (!costume) {
```

Reply:
I guess I don't really know when a null value would come up instead of undefined, so that seems reasonable. I did `undefined` because that is what the value was if the id passed into the function was wrong. The other reason was I thought being more specific with the data type versus using truthy or nullish would avoid a possible bug down the road if some code did ever cause the value being assessed to be null or an empty string, etc. for whatever reason. Perhaps that is premature optimization?

Reply:
In this case, since costume should be an object, any false-y value would represent an issue - I think it would probably actually be undefined, like you said, but this is a case where a truthy check is very common (same as in React, where we often get things from backend and will check if they are truthy to check if they are a defined object or not)

Reply:
Makes perfect sense. Thank you for clarifying!